### PR TITLE
Adding caribbean subfolder & barbados page to travel guide

### PR DESCRIPTION
**Pull request by sdm03**
Added caribbean subfolder in north america directory
Added barbados.md file to caribbean folder

This refers to issue #4 from UCL-RITS/rse-classwork-2020#4

### CODE REVIEW

File: north_america/README.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # NORTH_AMERICA
+
+Internal links:
+
+  - [caribbean/](./caribbean)
```
Comment (dpshelio) [inline north_america/README.md:5]:
```suggestion
  - [caribbean](./caribbean/README.md)
```

**Review by dpshelio — CHANGES_REQUESTED** [review]
small problem with the link

**Review by dpshelio — APPROVED** [review]
I've fixed the problem and now it can be included

**Comment by sdm03** [comment]
Thank you!

**Pull request merged**